react-app/src/components/UploadManager.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import { 
+  TextField,
+  IconButton,
+  Typography
+} from '@material-ui/core'
+import ArrowUpwardIcon from '@material-ui/icons/ArrowUpward'
+
+class UploadManager extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      name: ''
+    }
+
+    this.onInputChange = this.onInputChange.bind(this)
+  }
+
+  onInputChange(event) {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  onUploadClick = () => {
+    localStorage[this.state.name] = this.props.photo.dataUri
+    this.props.setPhotoTaken(false)
+  }
+
+  render() {
+    return (
+      <div style={{display: 'flex', padding: '12px 12px', alignItems: 'center'}}>
+        <span>
+          <label htmlFor='name'>
+            <Typography variant="h6" style={{padding: '0px 12px'}}>
+              Name:
+            </Typography>
+          </label>
+        </span>
+        <TextField
+          id='name'
+          name='name' 
+          type='text'
+          variant='outlined'
+          value={this.state.name}
+          onChange={this.onInputChange}
+        />
```

Comment:
For some reason this is giving suggestions based on previous input, which I don't think is super useful.

I don't think I've ever seen it do that, but it might be because of `type='text'`? So you may be able to just remove that.